core/require.js

```diff
@@ -66,9 +66,12 @@ module.exports = function () {
     return {
         visitor: {
             Program(path) {
-                path.unshiftContainer('body', babylon.parse('require = (global || GLOBAL).requireHook(require);').program.body[0]);
+                path.unshiftContainer("body", babylon.parse(module.exports.injectionString).program.body[0]);
             }
         }
     };
 };
+
+module.exports.injectionString = "require = (global || GLOBAL).requireHook(require);";
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/theprojectshop/Kassy/file/2737018411/issues/source?bid=3282987&fileBranchId=3347522#l75)